README.md

```diff
@@ -22,27 +22,32 @@ In this repository, annotations are placed in the `rbi/annotations` folder.
 
 Each annotation from the `rbi/annotations` folder must be defined in the `index.json` file at the root of this repository:
 
-```json
+```jsonc
 {
     "gemA": { // gem name
-        "dependencies": [ // optional: list of gems that need to be installed to test gemA RBI
+        // optional: list of gems that need to be installed to test gemA RBI
+        "dependencies": [
             "gemB",
             "gemC"
         ],
-        "requires": [ // optional: list of files that need to be required to test gemA RBI
+        // optional: list of files that need to be required to test gemA RBI
+        "requires": [
             "file1",
             "file2",
         ]
     }
 }
 ```
 
+If you're copying this into your own `index.json`, make sure you strip out the comments.
```

Comment:
Considering we have to specify this, doesn't it make sense to keep them as frightening red flashy strings in the README?